test/functional/connection/connection.ts

```diff
@@ -384,30 +387,39 @@ describe("Connection", () => {
     describe("Can change postgres default schema name", () => {
         let connections: Connection[];
         beforeEach(async () => {
-            connections = await createTestingConnections({ 
-                enabledDrivers: ["postgres"],
-                entities: [Post],
-                schemaName: "test-schema",
-                dropSchemaOnConnection: true
-            });
+            const [[connection1], [connection2]] = await Promise.all([
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [Comment, Guest],
+                    schemaName: "test-schema",
+                    dropSchemaOnConnection: true
+                }),
+                createTestingConnections({ 
+                    enabledDrivers: ["postgres"],
+                    entities: [Comment, Guest],
+                    schemaName: "another-schema",
+                    dropSchemaOnConnection: true
+                })
+            ]);
+            connections = [connection1, connection2];
         });
         afterEach(() => closeTestingConnections(connections));        
         it("schema name can be set", () => {
             return Promise.all(connections.map(async connection => {
                 await connection.syncSchema(true);
 
-                const post = new Post();
-                post.title = "ChangeSchemaName";
+                const comment = new Comment();
+                comment.context = "ChangeSchemaName";
 
-                const PostRepo = connection.getRepository(Post);
-                await PostRepo.persist(post);
+                const CommentRepo = connection.getRepository(Comment);
```

Comment:
please follow a common convention and naming it something like `repository` or `commentRepository`